core/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.8.0 (2024-02-16)
+
+- **Feature:** `WithCaptureHTTPResponse` was moved from `config` package to a new `runtime` package, to avoid confusion due to it not being a configuration option
```

Comment:
Add explicitly that `config.WithCaptureHTTPResponse` is deprecated and that `runtime.WithCaptureHTTPResponse` should be used instead, saying it was "moved" is not clear

Reply:
Lint was complaining because the latest `core` release doesn't have a `runtime` package, which the example was calling
And it also complains if it uses a deprecated function

So I'll have to do a release with the new method, and another with the old method deprecated and the example changed

Updated the changelog and added to the global changelog, please check